src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid $c-borderColor;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  color: $c-basicHeader;
+  font-size: $font-basic-size;
+  line-height: 14px;
+  
+  &__image {
+    display: flex;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__tittle {
+    text-align: left;
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: $font-basic-size;
+    font-weight: $roboto-medium;
+  }
+
+  &__review,
+  &__code,
+  &__price {
+    font-weight: 400;
+  }
+
+  &__review,
+  &__code {
+    font-size: 10px;
+  }
+
+  &__code,
+  &__price {
+    color: $c-specificGrey;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: $roboto-bold;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 0;
+    font-size: 14px;
+    text-align: center;
+    background: $c-specificBlue;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    text-transform: uppercase;
+    font-weight: $roboto-bold;
+    color: $c-white;
+  }
+
+  &__link:hover {
+    background: $c-white;
+    color: $c-specificBlue;
+    border-color: $c-specificBlue;
+  }
+  
+  &__block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    text-align: right;
+  }
+  
+  &__block--second {
+    margin-bottom: 16px;
+    line-height: 18px;
```

Comment:
make variable, not to use the same size in a few cases

Reply:
agree